Documentation/Contribute/UploadBinaryData.rst

```diff
@@ -6,202 +6,123 @@ Upload Binary Data
 
-1. The `Girder web interface`_.
-2. The `girder-cli` command line executable that comes with the
-   girder-client_ Python package.
-
-Before uploading data, please visit `data.kitware.com`_ and
-register for an account.
+1. The CMake ExternalData Web3 upload browser interface.
+2. The `w3` command line executable that comes with the
+   `@web3-storage/w3`_ Node.js NPM package.
 
-Once files have been uploaded to your account, they will be publicly
-available and accessible since data is content addressed. At release time,
-the release manager will upload and archive repository data references in the
-`ITK collection`_ and other redundant storage locations.
+Once files have been uploaded to your account, they will be publicly available
+and accessible since data is content addressed on the IPFS peer-to-peer
+network. At release time, the release manager will upload and archive
+repository data references in other redundant storage locations.
 
 Upload Via the Web Interface
 ----------------------------
 
+Use the `CMake ExternalData Web3 Upload`_ tool to upload your data to the
+InterPlanetary Filesystem and download the corresponding CMake content link
+file.
 
-.. figure:: LogInHighlighted.png
-  :alt: Log in welcome page
+.. figure:: CMakeW3ExternalDataUpload.png
+  :alt: CMake ExternalData Web3 Upload
   :align: center
-  :width: 400
+  :width: 500
+  :target: https://cmake-w3-externaldata-upload.on.fleek.co/
 
-  After logging in, you will be presented with the welcome page. Click on the
-  **personal data space** link.
-
-.. figure:: PersonalDataSpaceHighlighted.png
-  :alt: Personal data space
-  :align: center
-  :width: 400
-
-  Next, select the **Public** folder of your personal data space.
-
-.. figure:: PublicFolderHighlighted.png
-  :alt: Public folder
-  :align: center
-  :width: 400
-
-  Click the **green upload button**.
-
-.. figure:: UploadHereHighlighted.png
-  :alt: The Upload files dialog
-  :align: center
-  :width: 400
-
-  Click the **Browse or drop files** to select the files to upload.
-
-.. figure:: UploadHereFilesSelectedHighlighted.png
-  :alt: The Upload files dialog with files selected
-  :align: center
-  :width: 400
+  `CMake ExternalData Web3 Upload`_
 
-  Click **Start Upload** to upload the file to the server.
+Add the file to the examples repository in your example's directory. Next time
+CMake configuration runs, it will find the new content link. During the next
+project build, the data file corresponding to the content link will be
+downloaded into the build tree.
 
-Next, proceed to `Download the Content Link`_.
+Upload Via CMake and Node.js CLI
+--------------------------------
 
-Upload Via Python Script
-------------------------
+Install the `w3` CLI with the `@web3-storage/w3`_ `Node.js`_ package::
 
-A Python script to upload files from the command line, `girder-cli`, is
-available with the girder-client_ python package. To install it::
+  $ npm install --location=global @web3-storage/w3
```

Comment:
The code blocks are causing complaints:
https://open.cdash.org/viewBuildError.php?type=1&buildid=8271764

Related docs:
https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#literal-blocks
https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#directive-code-block
https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#directive-highlight

Reply:
Thanks for the pointers, @jhlegarreta !

Pushing another patch to address the complaints.

Reply:
:+1: builds are passing now. Thanks Matt.